src/popup-generation/htmlGenerators.js

```diff
@@ -41,10 +37,8 @@ const fieldContent = ({fieldName, fieldDisplayName, fieldType}) => {
 			return textOrNumberContent(fieldName, fieldDisplayName, true)
 		case FIELD_TYPES.number:
 			return textOrNumberContent(fieldName, fieldDisplayName)
-		case FIELD_TYPES.zillowLink:
-			return zillowLink()
-		case FIELD_TYPES.wikipediaLink:
-			return wikipediaLink()
+		case FIELD_TYPES.link:
+			return link(fieldName, fieldDisplayName)
```

Comment:
A little hacky, but to keep things fast/easy, I just used the variable `fieldName` to hold the URL, rather than try to refactor the code to specifically use a `url` field **just** in the case of a Link.